command/v7/shared/app_summary_displayer.go

```diff
@@ -165,6 +165,12 @@ func (display AppSummaryDisplayer) displayProcessTable(summary v7action.Detailed
 	if summary.Deployment.StatusValue == constant.DeploymentStatusValueActive {
 		display.UI.DisplayNewline()
 		display.UI.DisplayText(display.getDeploymentStatusText(summary))
+
+		var maxInFlight = summary.Deployment.Options.MaxInFlight
+		if maxInFlight > 0 && maxInFlight != constant.DeploymentMaxInFlightDefaultValue {
```

Comment:
*question:* Are we trying to future-proof this? we could just do `maxInFlight > constant.DeploymentMaxInFlightDefaultValue`

Reply:
Yes, in the sense that if `constant.DeploymentMaxInFlightDefaultValue` is set to 2 in the future, we wouldn't display max-in-flight if the user set `max-in-flight: 1`